pkg/k8s/pipelineRun.go

```diff
@@ -260,9 +264,9 @@ func (r *pipelineRun) GetSpec() *api.PipelineSpec {
 }
 
 // InitState implements part of interface `PipelineRun`.
-func (r *pipelineRun) InitState() error {
+func (r *pipelineRun) InitState(ctx context.Context) error {
 	r.ensureCopy()
-	klog.V(3).Infof("Set State to New [%s]", r.String())
+	klog.V(3).Info("Set state to 'new'")
```

Comment:
```suggestion
	klog.V(3).InfoS("Set state to 'new'")
```
Why don't you use InfoS here?

Reply:
Such logs that I missed initially to convert to structured (contextual) logs. It is already caught by [logcheck](https://github.com/kubernetes-sigs/logtools/tree/main/logcheck) tool, and I am migrating it to structured logging (WIP).

I am also planning to integrate logcheck tool into the PR voter to make sure we no more use non-structured logs.

Reply:
Log migration to structured log done - 5d0e992efedd8d2ffc589b9595bb5b0a8587160c.